### ISSUE TITLE
Postman issue

### ISSUE THREAD

**Issue by drussell-lacma** [DallasMuseumArt/OctoberFriends#4]
I installed the DMA.friends plugin and am receiving a: Class 'Postman' not found
error when trying to select "Friends Settings" from the admin menu. I pulled the source from github yesterday, 2/25/15
![issue2](https://cloud.githubusercontent.com/assets/11051047/6397866/a5b4f9b6-bd9c-11e4-96fd-a5f284a4d5ca.jpg)

**Comment by karnold** [comment]
what php version are you using?  And did you pull from the master branch or a specific release?

**Comment by drussell-lacma** [comment]
PHP 5.4.37. I pulled it from the "Download Zip" link on this page:
https://github.com/DallasMuseumArt/OctoberFriends

-Doug

On 2/26/2015 10:08 AM, Kristen Arnold wrote:

> what php version are you using? And did you pull from the master 
> branch or a specific release?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/DallasMuseumArt/OctoberFriends/issues/4#issuecomment-76231700.

## 

Douglas Russell
Director of Software Development and Application Support
Los Angeles County Museum of Art
www.lacma.org http://www.lacma.org
T 323.857.4747
E drussell@lacma.org mailto:drussell@lacma.org

**Comment by karnold** [comment]
And what version of octobercms are you running?

**Comment by drussell-lacma** [comment]
I'm not sure. I downloaded  and installed it on 2/20 from the master branch.

On 2/26/2015 12:01 PM, Kristen Arnold wrote:

> And what version of octobercms are you running?
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/DallasMuseumArt/OctoberFriends/issues/4#issuecomment-76257837.

## 

Douglas Russell
Director of Software Development and Application Support
Los Angeles County Museum of Art
www.lacma.org http://www.lacma.org
T 323.857.4747
E drussell@lacma.org mailto:drussell@lacma.org

**Comment by karnold** [comment]
This could possibly be the issue. you need to be running on build 186.  The specific commit is located here https://github.com/octobercms/october/commit/92ec1993302c82f7399c61518bfc6e2079e94496 https://github.com/octobercms/october/commit/92ec1993302c82f7399c61518bfc6e2079e94496

If you installed octobercms from git you can run the following

git checkout 92ec1993302c82f7399c61518bfc6e2079e94496
rm -Rf vendor
composer update

**Comment by drussell-lacma** [comment]
I didn't use git. How do you install from scratch?

Thanks,
Doug

On 2/26/2015 1:13 PM, Kristen Arnold wrote:

> This could possibly be the issue. you need to be running on build 186. 
> The specific commit is located here 
> https://github.com/octobercms/october/commit/92ec1993302c82f7399c61518bfc6e2079e94496 
> https://github.com/octobercms/october/commit/92ec1993302c82f7399c61518bfc6e2079e94496
> 
> If you installed octobercms from git you can run the following
> 
> git checkout 92ec1993302c82f7399c61518bfc6e2079e94496
> rm -Rf vendor
> composer update
> 
> > On Feb 26, 2015, at 3:09 PM, drussell-lacma 
> > notifications@github.com wrote:
> > 
> > I'm not sure. I downloaded and installed it on 2/20 from the master 
> > branch.
> > 
> > On 2/26/2015 12:01 PM, Kristen Arnold wrote:
> > 
> > > And what version of octobercms are you running?
> > > 
> > > —
> > > Reply to this email directly or view it on GitHub
> > > 
> > > https://github.com/DallasMuseumArt/OctoberFriends/issues/4#issuecomment-76257837.
> > 
> > ## 
> > 
> > Douglas Russell
> > Director of Software Development and Application Support
> > Los Angeles County Museum of Art
> > www.lacma.org http://www.lacma.org
> > T 323.857.4747
> > E drussell@lacma.org mailto:drussell@lacma.org
> > —
> > Reply to this email directly or view it on GitHub 
> > https://github.com/DallasMuseumArt/OctoberFriends/issues/4#issuecomment-76273951.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/DallasMuseumArt/OctoberFriends/issues/4#issuecomment-76274586.

## 

Douglas Russell
Director of Software Development and Application Support
Los Angeles County Museum of Art
www.lacma.org http://www.lacma.org
T 323.857.4747
E drussell@lacma.org mailto:drussell@lacma.org

**Comment by karnold** [comment]
I can confirm that this issue only creeps up in the latest version of octobercms.

Until we have an opportunity to update the plugin to support the latest build of octobercms you will need run off of the git version.  

We have started a new branch to support the latest version which you can watch here https://github.com/DallasMuseumArt/OctoberFriends/tree/october-rc-upgrade.  Once it is complete those changes will be merged into the master branch.

**Comment by carlos-arroyo** [comment]
This issue is resolved on [OctoberFriends/october-rc-upgrade](https://github.com/DallasMuseumArt/OctoberFriends/tree/october-rc-upgrade) branch
